drake/systems/plants/@Manipulator/parameterEstimation.m

```diff
@@ -34,17 +34,26 @@
 % @option model determines which model to use for estimation
 % 'dynamic'     = use dynamic model - requires qdd
 % 'energetic'   = use energetic model - doesn't require qdd
+% 'simerr'      = minimize simulation error
 % 
 % @option method determines which method to use for estimation
 % 'nonlinprog'    = nonlinear least squares (LS) to solve problem
 % 'linprog'       = linear LS on lumped params then nonlinear LS to recover
 %                 original parameters
-% 'simerr'      = minimize simulation error
-% 'lsqnonlin'   = use MATLAB's built-in nonlinear least squares solver (debugging)
+
+% % DEBUGGING VARIABLES
+% global numran 
+% numran = 0;
+% global TRAJFIG
+% global PERRFIG;global PERR;global PERRMIN
+% global PDIFFFIG;global PTRUE;global PDIFF
```

Comment:
These globals are purely for testing - yah, they shouldn't belong in master.  I put them in so I could show you some of the convergence behavior, but I really should have just kept them in my local branch.  I'll delete them.